MixpanelDemo/MixpanelDemoTests/MixpanelBaseTests.swift

```diff
@@ -22,12 +21,6 @@ class MixpanelBaseTests: XCTestCase, MixpanelDelegate {
         super.setUp()
         stubCalls()
         mixpanelWillFlush = false
-        
-        if let loginView = self.topViewController() as? LoginViewController {
```

Comment:
did you mean to get ride of this and make the storyboard changes as well?

Reply:
yeah, for tests, we remove the need depending on the demo app